mscore/musescore.cpp

```diff
@@ -3491,7 +3491,7 @@ static void loadScores(const QStringList& argv)
                                     score->setName(mscore->createDefaultName());
                                     // TODO score->setPageFormat(*MScore::defaultStyle().pageFormat());
                                     score->doLayout();
-                                    score->setCreated(true);
+                                    score->setEmpty(true);
```

Comment:
Am I correct this `empty` flag is never set to `false` then, even after adding something to a score?

Also the empty score is still newly created so perhaps the `setCreated(true)` call is still needed here.

Reply:
> Am I correct this empty flag is never set to false then, even after adding something to a score?

Yes, that's correct.

Rest assured that it's being handled correctly, but your observation makes it clear that I named this flag poorly.

How about `fromEmpty`, since it indicates that the score was created empty (as opposed to being created using the wizard or opened from a file)?

> Also the empty score is still newly created so perhaps the setCreated(true) call is still needed here.

No, I removed the call to `setCreated(true)` here because it was redundant (already unconditionally being done elsewhere).

Reply:
I went ahead and renamed `empty` to `startedEmpty` and also clarified the accompanying comment.